exotic/inputs.py

```diff
@@ -189,7 +189,7 @@ def comp_params(self, init_file, planet_dict):
 
 
 def check_imaging_files(directory, img_type):
-    file_extensions = ['.fits', '.fit', '.fts', '.fz']
+    file_extensions = ['.fits', '.fit', '.fts', '.fz', '.gz']
```

Comment:
hi @tamimfatahi  - should this line:
file_extensions = ['.fits', '.fit', '.fts', '.fz', '.gz']
be instead:
file_extensions = ['.fits', '.fit', '.fts', '.fz', 'fits.gz', 'fit.gz']
otherwise it might accept just any gzipped file…?